### PR TITLE
panflute 2.1.2 release

### PR DESCRIPTION
**Pull request by ickc**
Summary:

- document the support of pandoc 2.14
- add badges,
    - add info on distributed versions on GitHub Releases, conda-forge. Ideally, these 3 versions should be consistent. See <https://github.com/sergiocorreia/panflute/pull/193#issuecomment-860027299>.
    - add license

### CODE REVIEW

**Comment by ickc** [comment]
New release is not needed as it is just documenting the support of pandoc 2.14 with no code change.

**Comment by alerque** [comment]
Please bump the release anyway. I understand it isn't needed and downstream packages can bump their package release versions to cover this (as you suggest for anaconda and I've already done for Arch Linux) but there are two other gripes:

1. The current PyPi release is older than the latest GitHub tag. I realize this isn't a code change, but it's setting off update alerts to downstream distro packagers that are invalid.

2. It would be a good excuse to merge my PR #195 so that Arch can stop patching the setup file as part of the packaging.

In other words it's just the sanitary thing to do even if you *could* get away without a bump.

**Comment by ickc** [comment]
@alerque, I understand your points but those are separate issues, regarding the mismatch of the versions on GitHub Releases and PyPI, that's something only @sergiocorreia can fix. I'll open another issue about this.

**Comment by ickc** [comment]
I'm expanding the scope of the PR to be a new panflute release.

**Comment by ickc** [comment]
The force-push is just a rebase to latest master.

**Comment by ickc** [comment]
@sergiocorreia, could you review this?

This should be a minor release, documenting the support of pandoc 2.14 among minor things.

**Review by sergiocorreia — APPROVED** [review]
